oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -251,19 +321,64 @@ public GoogleCredentials(AccessToken accessToken) {
     this(accessToken, null);
   }
 
+  /**
+   * Constructor with using builder All the fields comes explicitly from builder.
```

Comment:
I think we can word this better (also "All" is incorrectly capitalized).

How about something like:

```
This constructor relies on a {@link GoogleCredential.Builder} to 
provide all the necessary field values for initialization
```

Reply:
Thanks for catching, seems like stopped in the middle of editing ) 

Suggestion sounds good